client/src/component/common/MainButton/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './style.css';
+import { Button } from 'antd';
+import PropTypes from 'prop-types';
+
+const MainButton = (props) => {
+  const { backgroundColor } = props;
+  const { border } = props;
+  const { color } = props;
+  const { padding } = props;
```

Comment:
You can destructuring the props like that.
```suggestion
const MainButton = ({backgroundColor, border, color, padding }) => {
```